### PR TITLE
download pages for netbeans 15

### PR DESCRIPTION
**Pull request by ebarboni**
uses of constants to ease for next release.

### CODE REVIEW

**Review by neilcsmith-net — APPROVED** [review]
Great idea!  Couple of minor changes.  Possibly get rid of the duplicate page at `nb15.html` and link everything via the index page?

File: netbeans.apache.org/src/content/download/nb14/nb14.asciidoc [hunk 1]
```diff
@@ -33,37 +33,48 @@ for important requirements for download pages for Apache projects.
...
+- Source: link:{url-status}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip[netbeans-{netbeans-version}-source.zip] (link:{fixed-download-url}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.sha512[SHA-512],
+link:{fixed-download-url}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.asc[PGP ASC])
 
 ////
 NOTE: Using https below is highly recommended.
 ////
 Officially, it is important that you link:https://www.apache.org/dyn/closer.cgi#verify[verify the integrity]
 of the downloaded files using the PGP signatures (.asc file) or a hash (.sha512 files).
-The PGP keys used to sign this release are available link:https://downloads.apache.org/netbeans/KEYS[here].
+The PGP keys used to sign this release are available link:{fixed-download-url}KEYS[here].
```
[the remark this answers is not in the document]
Comment (ebarboni) [reply netbeans.apache.org/src/content/download/nb14/nb14.asciidoc:77]: fixed download is https://download.apache.org like pgp and checksum

[the remark this answers is not in the document]
Comment (neilcsmith-net) [reply netbeans.apache.org/src/content/download/nb14/nb14.asciidoc:77]: Sorry, misread! :roll_eyes:

File: netbeans.apache.org/src/content/download/nb14/index.asciidoc [hunk 2]
```diff
@@ -33,37 +33,48 @@ for important requirements for download pages for Apache projects.
...
-link:https://downloads.apache.org/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-windows-x64.exe.asc[PGP ASC])
-* link:https://www.apache.org/dyn/closer.cgi/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-linux-x64.sh[Apache-NetBeans-14-bin-linux-x64.sh] (link:https://downloads.apache.org/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-linux-x64.sh.sha512[SHA-512],
-link:https://downloads.apache.org/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-linux-x64.sh.asc[PGP ASC])
-* link:https://archive.apache.org/dist/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-macosx.dmg[Apache-NetBeans-14-bin-macosx.dmg] (link:https://archive.apache.org/dist/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-macosx.dmg.sha512[SHA-512],
-link:https://archive.apache.org/dist/netbeans/netbeans-installers/14/Apache-NetBeans-14-bin-macosx.dmg.asc[PGP ASC])
+* link:{url-status}netbeans-installers/{netbeans-version}/Apache-NetBeans-{netbeans-version}-bin-windows-x64.exe[Apache-NetBeans-{netbeans-version}-bin-windows-x64.exe] (link:{fixed-download-url}netbeans-installers/{netbeans-version}/Apache-NetBeans-{netbeans-version}-bin-windows-x64.exe.sha512[SHA-512],
```
Comment (neilcsmith-net) [inline netbeans.apache.org/src/content/download/nb14/index.asciidoc:62]: Reverse of previous comment.  We have previously pointed SHA and PGP to the archive as well (until NB13). It does look like the download.a.o links for NB13 redirect automatically.  Not sure that has always been the case.

File: netbeans.apache.org/src/content/templates/news.gsp [hunk 3]
```diff
@@ -23,8 +23,8 @@
     <div class='grid-container'>
         <div class='cell'>
             <div class="annotation">Latest release</div>
-            <h1>Apache NetBeans 14</h1>
-            <p><a class="button success" href="/download/nb14/nb14.html">Download</a></p>
+            <h1>Apache NetBeans 15</h1>
+            <p><a class="button success" href="/download/nb15/nb15.html">Download</a></p>
```
[the remark this answers is not in the document]
Comment (ebarboni) [reply netbeans.apache.org/src/content/templates/news.gsp]: Should be ok (removed nb14.asciidoc)

File: netbeans.apache.org/src/content/download/nb14/index.asciidoc (hunk 2 above)
Comment (ebarboni) [reply netbeans.apache.org/src/content/download/nb14/index.asciidoc]: well github prune my comment, for KEYS hardcodeurl for pgp-checksum url-download-keychecksum https://archive.apache.org/dist/netbeans/  (//archived)
https://downloads.apache.org/netbeans/ (//current)
and for url-download
https://archive.apache.org/dist/netbeans/  (//archived)
https://www.apache.org/dyn/closer.cgi/netbeans/ (//current)

Comment (ebarboni) [reply netbeans.apache.org/src/content/download/nb14/index.asciidoc]: rebased for merging

**Review by geertjanw — APPROVED** [review]

**Comment by ebarboni** [comment]
lets merge

**Comment by ebarboni** [comment]
links ok on live site.

**Pull request merged**